py2hwsw/hardware/fpga/quartus/arria10_gx_dk/iob_arria10_gx_dk.py

```diff
@@ -2,25 +2,59 @@
 #
 # SPDX-License-Identifier: MIT
 
-bsp = [
-    {"name": "BAUD", "type": "M", "val": "115200"},
-    {"name": "FREQ", "type": "M", "val": "50000000"},
-    {"name": "IOB_MEM_NO_READ_ON_WRITE", "type": "M", "val": "1"},
-    {"name": "DDR_DATA_W", "type": "M", "val": "32"},
-    {"name": "DDR_ADDR_W", "type": "M", "val": "28"},
-    {"name": "INTEL", "type": "M", "val": "1"},
-]
-
 
 def setup(py_params_dict):
     attributes_dict = {
         "name": py_params_dict["instantiator"]["name"] + "_arria10_gx_dk",
         "generate_hw": True,
         "confs": [
             {
-                "name": "bsp",
-                "descr": "Board Support Package confs",
-                "confs": bsp,
+                "name": "AXI_ID_W",
+                "descr": "AXI ID bus width",
+                "type": "F",
+                "val": "1",
+            },
+            {
+                "name": "AXI_LEN_W",
+                "descr": "AXI burst length width",
+                "type": "F",
+                "val": "4",
+            },
+            {
+                "name": "AXI_ADDR_W",
+                "descr": "AXI address bus width",
+                "type": "F",
+                "val": "28",
+            },
+            {
+                "name": "AXI_DATA_W",
+                "descr": "AXI data bus width",
+                "type": "F",
+                "val": "32",
+            },
+            {
+                "name": "BAUD",
```

Comment:
why does it need to know this?

Reply:
I left those parameters in the generic fpga board wrappers so that the user can use them as a reference to create their own board wrappers.

For example, if the user wants to create a board wrapper for the `arria10_gx_dk` board, he can view that file to copy the values for the frequency, axi_addr_w (external memory size), mem_no_read_on_write, etc

Should these parameters be moved to a document instead? Maybe the py2hwsw user guide?